data/mods/Magiclysm/items/ethereal_items.json

```diff
@@ -744,6 +744,38 @@
       "passive_effects": [ { "has": "WIELD", "condition": "ALWAYS", "values": [ { "value": "ITEM_DAMAGE_COLD", "add": 6 } ] } ]
     }
   },
+  {
+    "id": "jar_3l_force",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "jar of force" },
```

Comment:
```suggestion
    "name": { "str": "jar of force", "str_pl": "jars of force" },
```

Reply:
Change is implemented. Thanks for pointing it out.